wenab-app/tools/generateClient.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const commander = require("commander");
```

Comment:
Why are we using JS lingo here over TS lingo? i.e. this instead of `import * as commander from "commander.js"` (or whatever the library name is)

Reply:
Oh Jesus... SO the imports are not JS vs TS. They are different module formats that node can be run in (ie: `CommonJS` vs `ES`). This is the only place that we use the `CommonJS` module format and it's because some library needed it at some point. I may be able to change it now but I don't want to 😅

You can read more about the different module formats at https://blog.logrocket.com/commonjs-vs-es-modules-node-js/